### ISSUE TITLE
Notifier les auteurs pour la publication d'un contenu

### ISSUE THREAD

**Issue by Arnaud-D** [zestedesavoir/zds-site#5745]
Actuellement, on envoie des MP pour différents événements de validation : réservation, mise à jour, refus, dépublication. Cependant, on ne le fait pas pour la publication tout court !

Il faudrait envoyer un petit MP automatique quand un staff valide et publie un contenu.

**Comment by HSleymn** [comment]
Tu as une proposition de résolution ?

**Comment by Arnaud-D** [comment]
Oui, ma proposition de résolution est décrite dans la PR #5940. C'est une grosse transformation de ce qui est fait actuellement pour remplacer les MP comme moyen de notification des événements sur les contenus, peu importe leur type (publication, réservation, ajouts d'auteurs, etc).

C'est en cours, même si c'est pas mal de travail, et actuellement l'avancement est ralenti parce que je dois retoucher progressivement du code un peu délicat d'autres modules pour rendre ma vision techniquement possible.

**Comment by Arnaud-D** [comment]
La PR qui ajoute le journal des événements est une étape désormais faite pour éviter d'utiliser les MP pour informer de la publication d'un contenu. Cependant, il n'y a pas actuellement de gestion des notifications associées, mais ça serait la meilleure solution plutôt qu'un MP comme suggéré ici.
